docs/sources/source/nlp/bert_pretraining.rst

```diff
@@ -7,7 +9,7 @@ Make sure you have ``nemo`` and ``nemo_nlp`` installed before starting this tuto
 The code used in this tutorial can be found at ``examples/nlp/language_modeling/bert_pretraining.py``.
 
 .. tip::
-    Pretrained BERT models can be found at 
+    Pretrained BERT models and model configuration files can be found at 
     `https://ngc.nvidia.com/catalog/models/nvidia:bertlargeuncasedfornemo <https://ngc.nvidia.com/catalog/models/nvidia:bertlargeuncasedfornemo>`__
```

Comment:
should we add these links to the intro page maybe in a separate section?

Reply:
ok